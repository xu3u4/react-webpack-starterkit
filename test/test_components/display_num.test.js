import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react'

import DisplayNum from 'components/display_num';

describe ('<DisplayNum>', () => {
  it ('render components', () => {
    const content = 3
    const { getByTestId } = render(
      <DisplayNum
        number={content}
        highlightClass="highlight"
      />
    );
    expect(getByTestId('number_block').textContent).toEqual("3");
  });
});