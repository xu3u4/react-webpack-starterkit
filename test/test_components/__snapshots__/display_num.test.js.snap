// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DisplayNum> render components 1`] = `
<div
  className="number_block highlight"
>
  10
</div>
`;
